plugins/modules/source_control/gitlab/gitlab_service.py

```diff
@@ -0,0 +1,467 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Raphaël Droz <raphael@droz.eu>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: gitlab_service
+short_description: Creates, updates or deletes GitLab services (aka "integrations").
```

Comment:
```suggestion
short_description: Creates, updates or deletes GitLab services (aka "integrations")
```
Please, also remove the acronym